build/release/dist.js

```diff
@@ -119,6 +119,7 @@ module.exports = function( Release, files, complete ) {
 		delete packageJson.devDependencies;
 		delete packageJson.dependencies;
 		delete packageJson.commitplease;
+		packageJson.version = Release.newVersion;
```

Comment:
Something I missed from an earlier PR

Reply:
How was this process working for previous releases then? jQuery `3.7.1` etc. has a correct version in its `package.json`.

Reply:
This was a regression from [today's PR](https://github.com/jquery/jquery/pull/5404) removing `scripts`.

Reply:
But why did it work previously? Before that PR, we weren't writing `package.json` to the disk at all in this script. Was this happening earlier by `jquery-release` itself and was the regression that we were overwriting a version already provided by `jquery-release`?

Reply:
It worked because jquery-release does change the version in the `package.json` and before that PR, the repo's package.json was copied to `dist`. Note that `pkg` here is read before the version is changed.

Reply:
Where was this copying happening? I don't see any code for that removed in https://github.com/jquery/jquery/pull/5404.

Reply:
It's only one line in the `extras` array.

Reply:
Ah, right… 🤦🏼‍♂️